.nais/nais-prod.yaml

```diff
@@ -39,4 +39,9 @@ spec:
     application:
       enabled: true
   secureLogs:
-    enabled: true
\ No newline at end of file
+    enabled: true
```

Comment:
Får ikke kommentert på riktig linje, men ser ut som at ingressen for prod peker til dev: https://amt-person-service.intern.dev.nav.no